test/test_um2netcdf.py

```diff
@@ -86,21 +86,23 @@ class DummyCube:
 
     def __init__(self, item_code, var_name=None, attributes=None,
                  units=None, coords=None):
-        self.item_code = item_code
+        self.item_code = item_code  # NB: um2nc adds this at runtime
         self.var_name = var_name or "unknown_var"
-        self.attributes = attributes
-        self.units = units
         self.standard_name = None
         self.long_name = ""
+
+        self.attributes = attributes or {}  # needs dict for coord()
         self.cell_methods = []
```

Comment:
Just wanted to check about the comment – Is `self.attributes` being a dictionary relevant for the `coord()` method?

Reply:
The change at line 105 calls `self.attributes.update`, so at that point it has to be a `dict`.

Reply:
Urgh, that's a typo thanks to lazily conflating `attributes` & the `coord` functionality.

Reply:
The comment is fixed in 53816cb0.